pages/auth/TEMPLATE.js

```diff
@@ -0,0 +1,34 @@
+import Head from "next/head";
```

Comment:
this is the template moving forward for any pages relating to the platform (ie: quizzes, assignments, lectures, etc...).  Copy and paste the code into whatever new page you are working on.

I created this template to make it easier for future page creations since every page will need some sort of authentication check -> we don't want someone not authorized to be able to access the site.